Created heisenberg theme for helix editor

IrishMaestro opened the pull request:
Dark theme inspired by Breaking Bad for helix.

Omnikar commented:
Could you provide a screenshot?

IrishMaestro commented:
Here are the screenshots. I also provided a screen recording to showcase the comments since I added a 'slow_blink' modifier.

![heisenberg_01](https://user-images.githubusercontent.com/70972101/195256099-01628ca1-50ed-4827-abcc-cf22e03b9b3e.png)
![heisenberg_02](https://user-images.githubusercontent.com/70972101/195256114-7eb66bf5-80ca-42f7-8cdd-439f523df523.png)

https://user-images.githubusercontent.com/70972101/195261031-f2d73b31-1e20-45a0-82f5-a345748f7287.mp4

the-mikedavis commented on runtime/themes/heisenberg.toml:
```diff
@@ -0,0 +1,80 @@
+"ui.linenr" = { fg = "crystal_blue" }
+"ui.linenr.selected" = { fg = "barium_green" }
+"ui.statusline" = { fg = "crystal_blue", bg = "black" } 
+"ui.cursorline" = { bg = "background" }
+"ui.popup" = { fg = "crystal_blue", bg = "background" }
+"ui.window" = { fg = "barium_green" }
+"ui.help" = { fg = "crystal_blue", bg = "background"}
+"ui.text" = { fg = "crystal_blue" }
+"ui.text.focus" = { bg = "background", fg = "barium_green" }
+"ui.text.info" = { fg = "crystal_blue" }
+"ui.virtual.whitespace" = { fg = "dark_gray" }
+"ui.virtual.ruler" = { bg = "background" }
```
Cursorline and rulers end up being invisible because they're set to the same color as the background. You need to enable config options these to see those two features: `:set cursorline true` and `:set rulers [80]`

IrishMaestro replied:
Good call. Updated `ui.cursorline` & `ui.virtual.ruler`

AlexanderBrevig commented:
Did you run `cargo xtask themelint heisenberg` ? :)

IrishMaestro commented:
> I see a few missing scopes. Please run `cargo xtask themelint heisenberg` and implement the missing scopes. Some of them may be accessible through config so you could enable those while authoring the theme :)
> 
> PS: cool theme and Breaking Bad is amazing!

Here is the initial result of the command listed above:

>heisenberg: `ui.selection` and `ui.selection.primary` cannot be equal
>heisenberg: `ui.statusline.normal` and `ui.statusline.insert` cannot be equal
>heisenberg: `ui.statusline.normal` and `ui.statusline.select` cannot be equal
>heisenberg: missing `bg` for `ui.cursorline.primary`
>heisenberg: missing `fg` or `bg` for `ui.virtual.indent-guide`
>Error: "heisenberg has issues"

I added colors for indentation & statusline modes (NOR, INS, SEL). I also resolved the warning regarding the missing background for `ui.cursorline.primary`.

What is the difference between `ui.selection` and `ui.selection.primary`? In the documentation listed on the website there is a note for `ui.selection`, but the section for `ui.selection.primary` is blank.

I noticed the visibilty on code blocks within README files was an issue as well. Changed the background to the theme's background for improved visibility.

Glad you enjoy the theme.

the-mikedavis commented:
Looks cool, thank you!

Could you add a screenshot of this to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

IrishMaestro commented:
> Looks cool, thank you!
> 
> Could you add a screenshot of this to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

Will do. Greatly appreciated
